src/pybind/mgr/devicehealth/module.py

```diff
@@ -220,6 +221,44 @@ def config_notify(self):
                     self.get_module_option(opt['name']))
             self.log.debug(' %s = %s', opt['name'], getattr(self, opt['name']))
 
+    def notify(self, notify_type, notify_id):
+       # create device_health_metrics pool if it doesn't exist
+       if notify_type == "osd_map":
+            if not self.device_pool:
+                self.create_pool()
+
+    def create_pool(self):
+        pools = self.rados.list_pools()
+        is_pool = False
```

Comment:
this variable name is confusing. what "is pool"?

Reply:
It was there before was just using the old snip.